File: VB/HowToSetValidationRulesInCode.Module.Win/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp.Updating
Imports DevExpress.ExpressApp
Imports DevExpress.Xpo

Namespace HowToSetValidationRulesInCode.Module.Win
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal objectSpace As ObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
		End Sub
	End Class
End Namespace

File: CS/HowToSetValidationRulesInCode.Module.Web/Updater.cs
using System;

using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;

namespace HowToSetValidationRulesInCode.Module.Web {
	public class Updater : ModuleUpdater {
		public Updater(Session session, Version currentDBVersion)
			: base(session, currentDBVersion) {
		}
		public override void UpdateDatabaseAfterUpdateSchema() {
			base.UpdateDatabaseAfterUpdateSchema();
		}
	}
}
